api/src/types/index.ts

```diff
@@ -1,46 +1,47 @@
-import * as prismic from "@prismicio/client";
-import { Client as ElasticClient } from "@elastic/elasticsearch";
-
-// Generic types
-export type ImageDimensions = {
-  width?: number;
-  height?: number;
+// TODO keep?
+import { ArticleFormatId, PrismicImage } from "../../../pipeline/src/types";
```

Comment:
I think we probably want to remove this dependency between the 2 apps, and actually remove all of the types here - on reflection, it probably wasn't the right call in the Concepts API to have the return types represented in TypeScript, as the `display` document is intended to be completely opaque JSON (ie the API doesn't need to know its type). In other words, we don't need the API to know what an `Article` is, but we do need the pipeline to know.